src/OpenLoco/Paint/PaintEntity.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include "../Types.hpp"
+
+namespace OpenLoco::Map
+{
+    struct map_pos;
+}
+
+namespace OpenLoco::Paint
+{
+    struct PaintSession;
+    void paintEntities(PaintSession& session, const Map::map_pos& loc);
+    void paintEntities2(PaintSession& session, const Map::map_pos& loc);
+}
```

Comment:
Do we really need this header? PaintEntity isn't really a separate thing from Paint.cpp, right?

Reply:
I've started to get a bit annoyed with lack of headers for some things. So i thought I'd start adding more of them. These functions are only used by paint and we don't need/want to expose them in the paint interface